### ISSUE TITLE
More details on browser support

### ISSUE THREAD

**Issue by betaorbust** [emotion-js/emotion#2697]
**Description:**
I'm trying to track down if Emotion will work for our (admittedly kind of sprawling) browser support matrix. 

In [the docs](https://emotion.sh/docs/introduction#browser-requirements), it says
> Emotion supports all popular browsers, including Internet Explorer 11.

Is there a source for what the minimums are for browsers like chrome, safari, mobile safari, firefox, etc? 

Thanks!

**Documentation links:**
https://emotion.sh/docs/introduction#browser-requirements

**Comment by Andarist** [comment]
Yes, it's hard to say what's the **exact** bottom range - but I can say that I know companies using Emotion that have really high expectations about browser support. So unless you have some really, really niche requirements then I can confidently say that you just shouldn't worry about it.

**Comment by betaorbust** [comment]
Haha we do have odd requirements, but I think the information you provided is good enough for me to keep exploring down my current path. 

> Out of curiosity, is IE support in Emotion important for your business? It's getting very hard to test in IE and I'd love to know if people are actually relying on support for that browser.

IE usage has dropped a lot in recent years but holdouts in S. Korea it's still registering as a couple % (a shadow left over from their legally-mandated use of IE for online purchases/banking for so many years) 
It will continue to fall off to the point where it's effectively gone, like it is elsewhere, but until then it's still a part of our support matrix.

Thanks for the help!

**Comment by Andarist** [comment]
At my previous job, we had to support IE11 too and Emotion was still working there just fine :)
